Mage.Verify/src/test/java/mage/verify/VerifyCardDataTest.java

```diff
@@ -1719,6 +1719,10 @@ private void compareClassRecursive(Object obj1, Object obj2, Card originalCard,
                     }
                 }
 
+                if (obj1 instanceof Ability) {
+                    checkAbility(originalCard, (Ability) obj1, msg);
```

Comment:
`compareClassRecursive` is a copy code checker. Use `checkMissingAbilities` instead and put special check in it. Example:

![shot_240521_172054](https://github.com/magefree/mage/assets/8344157/fc244ff2-b964-4538-aabd-6721d69084af)

Reply:
for that test to do anything, it needs to attempt to deep recurse down the abilities to find the triggers. `checkMissingAbilities` is doing text at high level.